assets/tests/components/tabBar.test.tsx

```diff
@@ -31,7 +31,7 @@ describe("tabBar", () => {
     const tree = renderer
       .create(
         <BrowserRouter>
-          <TabBar pickerContainerIsVisible={true} />
+          <TabBar pickerContainerIsVisible={true} swingsViewIsVisible={false} />
```

Comment:
could we make `swingsViewIsVisible` an optional prop, so that it defaults to `undefined` which is falsy, rather than add `swingsViewIsVisible={false}` everywhere?